src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/PhrasalTemplate/PhrasalTemplateField/PhrasalTemplateRenderField.tsx

```diff
@@ -146,9 +191,27 @@ export function RenderedField({
               }}
             />
           )}
+          {fromResponse?.items && (
+            <SelectController
+              name={`${name}.itemIndex`}
+              control={control}
+              defaultValue=""
+              fullWidth
+              options={fromResponse?.items.map(({ label }, index) => ({
+                labelKey: label as string,
+                tooltip: <StyledMdPreview modelValue={(name as unknown as string) ?? ''} />,
+                value: `${index}` ?? '',
```

Comment:
This nullish coalescing operation will never be used because a zero-length string never evaluates to nullish (it can be falsey, but never nullish), so you can remove it:
```suggestion
                value: `${index}`,
```